### ISSUE TITLE
StringSigils issue in 1.0.5

### ISSUE THREAD

**Issue by rodrigues** [rrrene/credo#654]
### Environment

* Credo version (`mix credo -v`): 1.0.5
* Erlang/Elixir version (`elixir -v`): 1.8.1
* Operating system: Mac OS Mojave

### What were you trying to do?

There's a parse error when interpolating in a capture operator / anonymous function.

Here it's an isolate repo with the issue: https://github.com/rodrigues/credo_capture_string/blob/master/lib/credo_capture_string.ex#L19

### Expected outcome

No parse error, as with 1.0.4

### Actual outcome
```
$ mix credo
Checking 2 source files ...
warning: Unexpected error while parsing lib/credo_capture_string.ex: [%Credo.Issue{category: :error, check: Credo.Code.ParserError, column: nil, exit_status: 0, filename: "nofilename", line_no: 26, message: {"unexpected 
token: ", ""}, meta: [], priority: 0, scope: nil, severity: nil, trigger: nil}]                              
lib/credo/check/readability/string_sigils.ex:50: Credo.Check.Readability.StringSigils.run/2
lib/credo/check/runner.ex:168: Credo.Check.Runner.do_run_check_on_single_source_file/3
lib/credo/check/runner.ex:161: Credo.Check.Runner.run_check_on_single_source_file/3
(elixir) lib/enum.ex:1327: Enum."-map/2-lists^map/1-0-"/2
(elixir) lib/enum.ex:1327: Enum."-map/2-lists^map/1-0-"/2

```

**Comment by liskin** [comment]
This is a bug in elixir tokenizer itself: https://github.com/elixir-lang/elixir/pull/8995

**Comment by TheFirstAvenger** [comment]
I was able to reproduce the issue in Elixir `1.8.1` but not in `1.9.0`. @rodrigues can you confirm it works for you in `1.9`?

**Comment by rodrigues** [comment]
@TheFirstAvenger indeed, it works fine for me in elixir 1.9.4, OTP 22.1.8

**Comment by liskin** [comment]
@TheFirstAvenger Well of course, https://github.com/elixir-lang/elixir/pull/8995 was merged in time for 1.9, but not backported to 1.8.

**Comment by TheFirstAvenger** [comment]
@rodrigues Good to hear. Could you close out this issue?
